sass/themes/donate/sr18/_donate-sr18-styles.scss

```diff
@@ -5,3 +5,48 @@ body.payin {
 .header {
   display: none;
 }
+
+/* Thank You page */
+#promo-header--success {
+  br {
+    display: inline;
+  }
+
+  h1, h2, h3 {
+    text-transform: none;
+  }
+
+  h2 {
+    margin-bottom: 10px;
+  }
+
+  .promo-header__content {
+    position: relative;
+    top: auto;
+    transform: translateY(0);
+
+    .promo-header__content-inner {
+      width: span(10);
+      padding: 30px 0 60px;
+    }
+  }
+ 
+  .thumb {
+    height: 144px;
+    width: 133px;
+    margin: 30px auto;
+    background-image: url('/images/form/thumb.png');
```

Comment:
since pulling master we can change the path now to use the image-path variable I think

Reply:
This is actually working fine, but i can update